dencam/buttons.py

```diff
@@ -86,9 +86,9 @@ def _handle_buttons(self):
                 self.latch_screen_button = True
 
                 self.state.goto_next()
-                if self.state.value == 4:
+                if self.state.value == self.STATE_LIST.index("BlankPage"):
                     self.recorder.start_preview()
-                elif self.state.value == 0:
+                elif self.state.value == self.STATE_LIST.index("NetworkPage"):
```

Comment:
This is also probably not quite right.  It is related to my comment on line 73.  Maybe the fix is to add a state at the beginning of STATE_LIST called something like "OffPage" ?  I haven't thought that through so try thinking it through and if it makes sense maybe do that and make the necessary change here and wherever else.